File: src/main/resources/codes/array.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.LinkedHashMap;
import java.util.List;

public class Main{
    public static void main(String[] args) {
        $(code)
        System.out.print(String.join(",",DataList.operations));
    }
}

class DataList {
    static ArrayList<String> operations = new ArrayList<>();
    List<Integer> arr;
    public DataList() {
        arr = Arrays.asList($(sample));
    }
    public Integer get(Integer idx) {
        operations.add("get" + "(" + idx + ")");
        return arr.get(idx);
    }
    public void swap(Integer i, Integer j) {
        int temp = arr.get(i);
        arr.set(i, arr.get(j));
        arr.set(j, temp);
        operations.add("swap" + "(" + i + "," + j + ")");
    }
    public Integer size() {
        return arr.size();
    }
}

File: src/main/resources/codes/array.py
class DataList:
    operations = []

    def __init__(self) -> None:
        self.arr = [$(sample)]

    def get(self, i):
        DataList.operations.append("get" + "(" + str(i) + ")")
        return self.arr[i]

    def swap(self, i, j):
        DataList.operations.append("swap" + "(" + str(i) + "," + str(j) + ")")
        temp = self.arr[i]
        self.arr[i] = self.arr[j]
        self.arr[j] = temp


$(code)
print(":".join(DataList.operations),end="")
